Backport for changes in postgres 11.4

MannerMan opened the pull request:
Backported to apply cleanly on postgres_fdw.c from latest postgres
11.4 point-release which contains changes to postgres_fdw.c causing
the previous patch not to apply.

ghost commented:
Thanks for the PR. Instead of changing the original patch I would rather have patches for different PSQL versions. Could you provide us with a `postgres_fdw_parallel_safe_11_4.patch` file?

oscarc-fnx committed:
Backport for changes in postgres 11.4

Backported to apply cleanly on postgres_fdw.c from postgres 11.4
point-release which contains changes to postgres_fdw.c causing
the previous patch not to apply.

MannerMan commented:
> Thanks for the PR. Instead of changing the original patch I would rather have patches for different PSQL versions. Could you provide us with a `postgres_fdw_parallel_safe_11_4.patch` file?

Indeed, that makes sense. I have pushed such change to this branch!

ghost commented:
Thanks!
